src/mixins/index.ts

```diff
@@ -1,7 +1,11 @@
+import CommonMixin from './common-mixin'
+import CurrencyLookupMixin from './currency-lookup-mixin'
```

Comment:
So currency-lookup-mixin is not exported/published ?

Reply:
I don't think we've published any mixins yet? 
Is that something we should do? 
Do you feel CurrencyLookup would be a good one to use elsewhere?

Reply:
I think you're right. We only have some partial mixins in here.

I think, ultimately, it would make sense to publish some mixins (ie, useful code that multiple apps could use). But it will depend on what they depend on (prob no store, depends on imports, etc).

As for CurrencyLookup, I guess you're only using it here, right? I'd be tempted to move it into the component's folder to keep it close to where it's used (and the whole thing gets published together).

Or it could the first mixin (or is that module -- see corp-type-module) that is published.

Currently, does this mixin have to exist in the app?

Reply:
I feel like we should publish it, if we find we need it.
Yes, i believe we need this mixin, unless you would prefer to see that function within the component itself. 
I prefer the mixin 🤷

Reply:
I was thinking this mixin could be moved into the component's folder (which, I know, isn't something we've done before, but this feels right here).

Up to you... wfm!

Reply:
okie doke, will relocate 👍 
Updated in next commit.